_inc/client/plans/single-product-backup.jsx

```diff
@@ -155,15 +172,70 @@ function PlanRadioButton( {
 	);
 }
 
-// eslint-disable-next-line no-unused-vars
+function ProductSavings( { selectedBackup, currencyCode } ) {
+	if ( ! selectedBackup ) {
+		return null;
+	}
+	const savingsCurrencyObject = getCurrencyObject( selectedBackup.potentialSavings, currencyCode );
+	const savings = formatCurrency( savingsCurrencyObject );
+
+	return __( 'You are saving %(savings)s by paying yearly', {
+		args: { savings },
+	} );
+}
+
+function UpgradeButton( { selectedUpgrade, billingTimeFrame, currencyCode, onClickHandler } ) {
+	if ( ! selectedUpgrade ) {
+		return null;
+	}
+	const { link, name, fullPrice, discountedPrice, type } = selectedUpgrade;
+	let billingTimeFrameString = '';
+	const timeframe = <span />;
+	const price = (
+		<PlanPrice currencyCode={ currencyCode } rawPrice={ discountedPrice || fullPrice } inline />
+	);
+	if ( 'yearly' === billingTimeFrame ) {
+		billingTimeFrameString = __( '{{price/}} {{timeframe}}per year{{/timeframe}}', {
```

Comment:
I'm not sure we should have that in a single string. In other places (Calypso), we always keep the price separate, and then we only have to translate `per year` and `per month`, and I think they already have translations.

This would also resolve another problem, where we have duplicate strings where only the timeframe is different.